README.md

```diff
@@ -39,12 +39,64 @@ limitations under the License.
     </a>
 </p>
 
-# Layer
+## Layer - Metadata Store for Production ML
+[Layer](https://layer.ai) helps you build, train and track all your machine learning project metadata including ML models and datasets with semantic versioning, extensive artifact logging and dynamic reporting with local↔cloud training
 
-Layer helps you create production-grade ML pipelines with a seamless local↔cloud transition while enabling collaboration with semantic versioning, extensive artifact logging and dynamic reporting.
-
-[Start for Free!](https://app.layer.ai)
+**[Start for Free now!](https://app.layer.ai/login?returnTo=%2Fgetting-started)**
 
 ## Getting Started
 
-The simplest way to get started with Layer is to go through the [quickstart guide](https://docs.app.layer.ai/docs/getting-started).
+Install Layer:
+```shell
+pip install layer --upgrade
+```
+
+Login to your free account and initialize your project:
+```python
+import layer
+layer.login()
+layer.init("my-first-project")
+```
+
+Decorate your training function to register your model to Layer:
+```python
+from layer.decorators import model
+
+@model("my-model")
+def train():
+    from sklearn import datasets
+    from sklearn.svm import SVC
+    iris = datasets.load_iris()
+    clf = SVC()
+    clf.fit(iris.data, iris.target)
+    return clf
+
+train()
+```
+
+Now you can fetch your model from Layer:
+
+```python
+import layer
+
+clf = layer.get_model("my-model:1.1")
```

Comment:
this should be `layer.get_model("my-model:1.1").get_train()` to return `SVC()`.

Reply:
Good catch. Fixed